features/fixtures/handled/browserify/src/a.js

```diff
@@ -0,0 +1,13 @@
+var bugsnag = require('bugsnag-js')
+var config = require('./lib/config')
+
+var bugsnagClient = bugsnag(config)
+
+bugsnagClient.notify(new Error('bad things'), {
+  beforeSend: function () {
+    setTimeout(function () {
+      var el = document.getElementById('bugsnag-test-state')
+      el.textContent = el.innerText = 'DONE'
+    }, 5000)
```

Comment:
It would be good to extract the `5000` from these scripts to one place, in case we decide to change it in future. Is this the delay required to send a report to Bugsnag?

Reply:
Having it duplicated means that each e2e test fixture is self-contained, making it easier to follow and and reduces the chance that a slight change in a helper library could break/invalidate the test result.